### DISCUSSION TITLE
Welcome to Spectral Discussions!

### DISCUSSION THREAD

**Discussion by philsturgeon** [stoplightio/spectral#1397] [General]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cot5r** [comment]
Looking to learn about this API protocol. I'm Bob. I'm relearning the basics of APIs & the design-first approach.
